.pre-commit-config.yaml

```diff
@@ -12,31 +12,30 @@ repos:
     -   id: check-docstring-first
 
 -   repo: https://github.com/psf/black
-    rev: 22.1.0
+    rev: 22.6.0
     hooks:
     -   id: black
         args: [--line-length=79]
 -   repo: https://github.com/PyCQA/flake8
-    rev: 4.0.1
+    rev: 5.0.4
     hooks:
     -   id: flake8
 -   repo: https://github.com/asottile/reorder_python_imports
-    rev: v2.7.1
+    rev: v3.8.2
     hooks:
     -   id: reorder-python-imports
         args: [--application-directories, '.:src', --py36-plus]
 -   repo: https://github.com/asottile/pyupgrade
-    rev: v2.31.0
+    rev: v2.37.3
     hooks:
     -   id: pyupgrade
         args: [--py36-plus]
 -   repo: https://github.com/asottile/add-trailing-comma
-    rev: v2.2.1
+    rev: v2.2.3
     hooks:
     -   id: add-trailing-comma
         args: [--py36-plus]
-- repo: https://gitlab.com/iamlikeme/nbhooks
-  rev: 1.0.0
-  hooks:
-  - id: nb-ensure-clean
-    args: [--meta, ExecuteTime]
+-   repo: https://github.com/kynan/nbstripout
+    rev: 0.6.0
+    hooks:
+    -   id: nbstripout
```

Comment:
Does this automatically clean the output? Do we want this? Do we not also want to allow to publish notebooks with the expected output present?

Reply:
IMHO the information of the old precommit hook that the notebook still contains output is sufficient. One can then decide to still keep it, or manually strip it. Just my 5 cent.

Reply:
yes, it does. This is typical way of sharing a notebook, as not all renders work in the same way.